.travis.yml

```
@@ -12,7 +12,7 @@ script:
     - yapf --diff --recursive onlinejudge setup.py tests | tee yapf.patch && test ! -s yapf.patch
     - mypy onlinejudge setup.py tests
     - make -C docs html
-    - pytest tests/*.py -v
+    - pytest tests/*.py -v --cov-report term-missing --cov=onlinejudge tests/
```

```suggestion
    - pytest tests/*.py -v --cov-report term-missing --cov=onlinejudge
```

最後の `tests/` が余分な気がします (たぶん)